tests/hello-world.js

```diff
@@ -3,22 +3,58 @@ import { check } from 'k6';
 import { Kubernetes } from 'k6/x/kubernetes';
 import * as deploy from "./common/common.js";
 import { sleep } from 'k6';
+import exec from 'k6/execution';
+
+const testName = "hello-world";
+// TODO: get this list from the scenario Option
+const cases = ["rust", "go", "js", "py"];
+const route = "hello"
+const replicas = 1;
+const namespace = `${__ENV.NAMESPACE}` != "undefined" ? `${__ENV.NAMESPACE}` : "default";
+let executor = `${__ENV.EXECUTOR}` != "undefined" ? `${__ENV.EXECUTOR}` : "containerd-shim-spin";
+let repo = `${__ENV.REPO}` != "undefined" ? `${__ENV.REPO}` : "ghcr.io/kate-goldenring/performance";
+let tag = `${__ENV.TAG}` != "undefined" ? `${__ENV.TAG}` : "latest";
 
 export let options = {
-  stages: [
-    { target: 200, duration: '10s' },
-    { target: 0, duration: '10s' },
-  ],
-  noConnectionReuse: true
+  tags: {
+    test: testName,
+  },
+  noConnectionReuse: true,
+  discardResponseBodies: true,
```

Comment:
Ditto, this is the cause for the failed checks re: res.body.  Alas, looks like we are getting a false positive from this test (testrun says succeeded even though the check failed) eg:

```
     ✗ body message was 'Hello, World'
      ↳  0% — ✓ 0 / ✗ 10263
```

Looks like we'll need to configure thresholds to properly fail a test when a given check fails: https://k6.io/docs/using-k6/thresholds/#fail-a-load-test-using-checks